src/entities/partner-access.entity.ts

```diff
@@ -1,20 +1,19 @@
-import { PartnerAdminEntity } from 'src/models/partner-admin.entity';
-import { PartnerEntity } from 'src/models/partner.entity';
-import { UserEntity } from 'src/models/user.entity';
-import { Column, Entity, ManyToOne, OneToOne } from 'typeorm';
+import { PartnerAdminEntity } from 'src/entities/partner-admin.entity';
+import { PartnerEntity } from 'src/entities/partner.entity';
+import { UserEntity } from 'src/entities/user.entity';
+import { Column, Entity, JoinColumn, ManyToOne, OneToOne } from 'typeorm';
 import { BaseEntity } from './base.entity';
 
 @Entity()
 export class PartnerAccessEntity extends BaseEntity {
   @OneToOne(() => UserEntity, (userEntity) => userEntity.partnerAccess)
-  user?: UserEntity;
+  @JoinColumn()
+  user: UserEntity;
 
-  @ManyToOne(() => PartnerEntity, (partnerEntity) => partnerEntity.partnerAccess, {
-    eager: true,
-  })
+  @ManyToOne(() => PartnerEntity, (partnerEntity) => partnerEntity.partnerAccess)
   partner: PartnerEntity;
 
-  @ManyToOne(() => PartnerAdminEntity, (partnerAdminEntity) => partnerAdminEntity.partnerAccess)
+  @ManyToOne(() => PartnerAdminEntity, (partnerAdminEntity) => partnerAdminEntity.admin)
   createdBy: PartnerAdminEntity;
```

Comment:
I'm finding these namings misleading. If `partnerAdminEntity.admin` was`partnerAdminEntity.partnerAccess` and `PartnerAccess.createdBy` was `PartnerAccess.partnerAdmin`, it'd be immediately obvious that it was a relationship to the other model. I thought the `createdBy` could stay but now we can see it two ways i think it needs to change